lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexMasterConf.java

```diff
@@ -67,6 +67,13 @@
   }
 
   /**
+   * Size of threadpool for callbacks on {@link org.apache.reef.vortex.api.VortexFuture}.
+   */
+  @NamedParameter(doc = "Size of threadpool for callbacks on VortexFuture.", default_value = "10")
+  final class CallbackThreadpoolSize implements Name<Integer> {
```

Comment:
Nit: We use capital P elsewhere (e.g., `ThreadPool`), so it would be better to be `CallbackThreadPoolSize`.